### PR TITLE
Sidebar toggle hotkey changes

### PR DESCRIPTION
**Pull request by loldudester**
(New pr because I am a dingus with git)

Adds a config option for the existing Ctrl+F11 hotkey which hides the sidebar entirely.

Also adds a new configurable hotkey that toggles the current/last open plugin panel (defaults to Ctrl+F12).

Both configs are at the bottom of the Runelite config panel.

java_tyLDQy8Uuk

### CODE REVIEW

**Comment by Adam-** [comment]
Can you split this into two commits? One for the sidebar key bind configuration & the other for the new panel toggle

File: runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java [hunk 1]
```diff
@@ -770,6 +779,29 @@ private void toggleSidebar()
...
+		if (isPanelOpen)
+		{
+			contract();
+		}
+		else
+		{
+			// Try to restore last panel
+			expand(currentNavButton);;
+
+			if (!container.isAncestorOf(pluginToolbar))
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java]: what is this for?

Comment (loldudester) [reply runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java]: Without this, if you use the new hotkey to open the plugin panel, after closing the whole sidebar with the old hotkey, you get the plugin panel *without* the actual narrow sidebar.

File: runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java [hunk 2]
```diff
@@ -770,6 +779,29 @@ private void toggleSidebar()
...
+		sidebarOpen = !sidebarOpen;
+
+		if (isPanelOpen)
+		{
+			contract();
+		}
+		else
+		{
+			// Try to restore last panel
+			expand(currentNavButton);;
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java]:
```suggestion
			expand(currentNavButton);
```

**Pull request closed without merging**